kool.yml

```diff
@@ -2,7 +2,6 @@ scripts:
   # Helper for local development - parsing presets onto Go,
   # compiling and installing locally
   dev:
-    - kool run parse-presets
```

Comment:
really wanted to remove this one?

Reply:
yeah, people are using it to compile/install locally only under testing environment... the parse-presets is only meant when we update presets so, went for removing from `dev`